xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailMetaSheetView.swift

```diff
@@ -48,6 +48,36 @@ struct MemoViewerDetailMetaSheetView: View {
                         Divider()
                         
                         if let address = store.state.address {
+                            if store.state.liked {
+                                Button(
+                                    action: {
+                                        store.send(.requestUpdateLikeStatus(address, liked: false))
+                                    },
+                                    label: {
+                                        Label(
+                                            "Remove from likes",
```

Comment:
I'd like to tighten up the copy:

- Like/Unlike
- Quote/Unquote

Un-verb is getting a little creative with language, but I think its worth it for the visual simplicity of seeing a single verb.